tests/test_sequence_manager.py

```diff
@@ -43,7 +43,7 @@ def test_sequence_manager_basics(mode: str):
     assert shutdown_evt.is_set()
 
 
-class TestSequenceManager(RemoteSequenceManager):
+class RemoteSequenceManagerWithChecks(RemoteSequenceManager):
```

Comment:
pytest thinks that classes whose name starts with `Test` are test cases and raises warnings during test collection. Fixing that.